provisioner/object-bucket-reconciler/reconciler.go

```diff
@@ -13,7 +13,7 @@ type ObjectBucketReconciler struct {
 
 // TODO if we decide that OBs should have their own Reconiler, then we can work out
 //  the logic for that here.  If not, this package can be deleted.
-func (r *ObjectBucketReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r ObjectBucketReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
removing ptr implies that `Reconcile()` does not modify its receiver in any way. Another reason to keep the `*` is if the receiver struct is somewhat large. Though passing by value may be correct, I see k8s code where all methods on a type are pointers rather than values even though receiver is not modified by some of the methods. Not sure if this consistency pays off in the long run?

Reply:
right, that's a typo, will add.